services/shhext/mailservers/connmonitor.go

```diff
@@ -0,0 +1,85 @@
+package mailservers
+
+import (
+	"sync"
+	"time"
+
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/p2p/enode"
+	"github.com/status-im/whisper/whisperv6"
```

Comment:
```suggestion
	whisper "github.com/status-im/whisper/whisperv6"
```

Reply:
tbh i don't like this at all. the package is whisperv6 and all tools are importing it like that by default

Reply:
We have been doing it this way so far so for consistency I would stick to `whisper "github.com/status-im/whisper/whisperv6"`. 

That's a default behaviour because these tools are unaware of the fact that whisper module is poorly versioned. It should be "github.com/status-im/whisper/v6" I believe and it would be seen as `whisper`.